transbank/common/integration_type_host_helper.py

```diff
@@ -0,0 +1,16 @@
+from transbank.common.integration_type import IntegrationType
+
+
+class IntegrationTypeHostHelper(object):
+    @classmethod
+    def webpay_host(cls, integration_type: IntegrationType) -> str:
```

Comment:
In python you are not forced to put helper methods/functions inside classes as static/class methods. They can live on their own as just functions. So you can just define `webpay_host` in the module and then import it like `from transbank.common.integration_type_host_helper import webpay_host`. 

BTW, I would just put the `webpay_host` method inside the `integration_type` module directly, so it would be `from transbank.common.integration_type import IntegrationType, webpay_host`

Reply:
webpay_host moved to module integration_type and helper class remvoved